src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTabel';
+
+type Props = {
+  slugPerson: string | undefined,
+};
+
+export const PeoplePage: React.FC<Props> = ({ slugPerson }) => {
+  const [people, setPeople] = useState<Person[]>();
+  const [error, setError] = useState<boolean>();
+  const [isLoading, setIsLoading] = useState<boolean>();
```

Comment:
I remember your questions in fe_chat.
It looks strange behavior of linter.
useState needs initialValue.
maybe try to  reinstall node modules

- remove node_modules `rm -rf node_modules`
- clean cash  `npm cache clean --force`
-  `node_modules npm install`

Reply:
Yes, now it is working.